sdk/Lusid.Sdk/Utilities/ClientCredentialsFlowTokenProvider.cs

```diff
@@ -63,26 +67,47 @@ public ClientCredentialsFlowTokenProvider(ApiConfiguration configuration)
         /// <inheritdoc />
         public async Task<string> GetAuthenticationTokenAsync()
         {
-            await _semaphore.WaitAsync();
-            try
+            AsyncRetryPolicy _policy;
```

Comment:
`_` are typically convention for class level variables rather than local method scoped ones